feat(content): New Alien Faction: The Gegno (#6610)
endless-sky/endless-sky: data/gegno/gegno missions.txt

mission "First Contact: Gegno"
	landing
	source "Giaru Gegno"
	to offer
		has "First Contact: Quarg: offered"
	on offer
		log "Factions" "Gegno" `The Gegno are a humanoid species to the galactic east of the core, split amongst themselves into branches. One branch is the Gegno Vi, who are spoken of to be strong and forceful brutes. The other is the Gegno Scin, mentioned to be progressers of research and technology. Apart from them is a complacent civilian population.`
		conversation

> 			"After jumping through several systems of hostile alien ships, a Quarg ring world is a sight of relief to you. From the looks of it, this ringworld is much newer than that of the one in human space, on top of being completed. Millions of Quarg probably inhabit this ring, but what's more interesting to your eye is that some of the aliens also are present. They are also humanoid, and the most common figures you come across are either slightly above or below an average human's height. Their skin is a smooth, light gray, adorned with different sorts of leather that remind you of medieval-style human clothing."

Author: This actually reminds me- part of the idea is that the gegno occupy planets and territory of a much older long gone race- will have to find a way to fit that into the dialog if necessary.

> 			"After jumping through several systems of hostile alien ships, a Quarg ring world is a sight of relief to you. From the looks of it, this ringworld is much newer than that of the one in human space, on top of being completed. Millions of Quarg probably inhabit this ring, but what's more interesting to your eye is that some of the aliens also are present. They are also humanoid, and the most common figures you come across are either slightly above or below an average human's height. Their skin is a smooth, light gray, adorned with different sorts of leather that remind you of medieval-style human clothing."

Author: Some planet dialogs hint to this as well, but won't be accessible until later on. Perhaps I should add the idea of such an old ringworld being questionable for. anew species to the player

> 			"After jumping through several systems of hostile alien ships, a Quarg ring world is a sight of relief to you. From the looks of it, this ringworld is much newer than that of the one in human space, on top of being completed. Millions of Quarg probably inhabit this ring, but what's more interesting to your eye is that some of the aliens also are present. They are also humanoid, and the most common figures you come across are either slightly above or below an average human's height. Their skin is a smooth, light gray, adorned with different sorts of leather that remind you of medieval-style human clothing."

Reviewer: If it's the case that it's an older ring being repurposed for the Gegno-watching Quarg, then would just need to remove the mentions of the ring being newer than the one in human space (I don't think the player character could judge that to be the case anyway, since I don't imagine ringworlds show many signs of the passage of time if there are Quarg living in them keeping them pristine as one would expect).
Other than that, the population mentions should be scaled up since this is a full ring, and there'd be a lot of Quarg in here (if it's an older ring, would for sure have their trillions pop, as Alta Hai alludes).

> 			`	The Quarg here move in a more eerily-quiet manner than what you're used to, almost dismissive to the other species walking about, who are equally as dismissive of you. The unusual atmosphere around you is finally broken when a few Quarg start to advance towards you quicker than normally. After a few minutes of unidentifinable discussion and gestures between themselves, one of them motions you to follow them.`

Author: Going to change it to moment. And yes, the idea was that they hurried over faster than the others, so applying your suggestion.

> 			`	You walk with the Quarg, occasionally having trouble keeping up to pace with their silently brisk steps, as you head towards a section of the ring that none of the aliens are at. Eventually, you are led to a smaller room with two more Quarg in waiting, silently sat down at the end of it. They talk, but this time you can hear a few familiar sounding terms, one being "Humani". Despite speaking to each other for what you can only assume has been half an hour, the Quarg still look at each other in bewilderment.`

Author: I think just "pace" is the right grammar. Changing it to that. 

Good suggestions; committing those. As for the time, going to knock this one down to minutes.

> 			`	One of the Quarg finally turns towards you and motions you to remain where you are for a little longer, offering you a seat. Their look of confusion disappears when, what sounded like a very faint, low-toned sigh, comes from them focusing on you again. Meanwhile, the other Quarg that was sat down has left the room, along with the ones that first brought you here.`

Author: Updated.

			choice
				`	"The Gegno?"`
					goto gegno
				`	"Are the ships I’ve seen these Gegno?"`
					goto ships
			label gegno
			`	"Yes, the imperious Gegno. A juvenile race, they are, many decades younger than that of you humans. They do not find themselves in the struggle of leadership the same way as others do. Rather, they glorify their pursuit of guidance between one another to share a noble future."`
			choice 
				`	"What is this guidance they pursue?"`
					goto guidance
			label ships
			`	"Indeed, the vessels of such stature represent the robust Gegno. Not in sorrow, but greatness do they pursue their future, bearing the fruit of sacrifice as a trophy to their name." The Quarg now shifts facial expressions in such a way to suggest a small hint of frustration."`
			choice

> 				`	"Why do they pursue in the name of sacrifice?"`

Reviewer 2: The previous line did not suggest to me to ask this question. I'm not even sure this question makes sense, either in general or in relation to the previous line.

Author: Would it be better for the player to ask what has made the Quarg frustrated? That may be too much of an assumption, but I'm thinking about what can lead them to the next bit.

					goto sacrifice
			label guidance

> 			`	The Quarg now shifts facial expressions in such a way to suggest a small hint of frustration. "The Gegno believe it to be necessary to sacrifice whatever must be, in order to imagine a future for themselves. Thus, they have separated into two different branches. One brandishes themselves as the honorable Vi- mightiest of warriors, toughest of kin. They are brutes that see to proof of strength being the one and only variable in determining their path. That of the Scin, the more technologically able, believe that only through progression of sciences and discovery, will the Gegno prosper."`
> 			label sacrifice
> 			`	"The Gegno believe it to be necessary to sacrifice whatever must be, in order to imagine a future for themselves. Thus, they have separated into two different branches. One brandishes themselves as the honorable Vi- mightiest of warriors, toughest of kin. They are brutes that see to proof of strength being the one and only variable in determining their path. That of the Scin, the more technologically able, believe that only through progression of sciences and discovery, will the Gegno prosper."`
> 
> 			choice

Reviewer 2: ```suggestion
			`	The Quarg now shifts facial expressions in such a way to suggest a small hint of frustration. "The Gegno believe it to be necessary to sacrifice whatever must be, in order to imagine a future for themselves. Thus, they have separated into two different branches. One brandishes themselves as the honorable Vi- mightiest of warriors, toughest of kin. They are brutes that see to proof of strength being the one and only variable in determining their path. That of the Scin, the more technologically able, believe that only through progression of sciences and discovery, will the Gegno prosper."`
				goto next
			label sacrifice
			`	"The Gegno believe it to be necessary to sacrifice whatever must be, in order to imagine a future for themselves. Thus, they have separated into two different branches. One brandishes themselves as the honorable Vi- mightiest of warriors, toughest of kin. They are brutes that see to proof of strength being the one and only variable in determining their path. That of the Scin, the more technologically able, believe that only through progression of sciences and discovery, will the Gegno prosper."`

			label next
			choice
```

				`"The Vi are right, the strength to survive is more important than anything else." `
					goto vi
				`"The Scin are right, technological advancement opens up so many possibilities."`
					goto scin
				`"Why do they not combine their efforts to better their future together?"`
					goto together

> 			label vi
> 			`	"While it is characteristic, such a tremendous proposition is rather dull for a growing species. Be that as it may, you humans follow to a much lesser degree than that of the Vi."`
> 			label scin
> 			`	The Quarg now shifts facial expressions in such a way to suggest a small hint of disappointment. "The pursuit of technology leaves behind many values important to a species' growth."`
> 			label together
> 			`	"Regrettably, the Gegno chiefly do not have a drive to pursue their future, diluting themeselves far into their present. Segregation of followings was one of few ways to seek motivation beyond the fact."`
> 			choice

Reviewer 2: ```suggestion
			label vi
			`	"While it is characteristic, such a tremendous proposition is rather dull for a growing species. Be that as it may, you humans follow to a much lesser degree than that of the Vi."`
				goto end
			label scin
			`	The Quarg now shifts facial expressions in such a way to suggest a small hint of disappointment. "The pursuit of technology leaves behind many values important to a species' growth."`
				goto end
			label together
			`	"Regrettably, the Gegno chiefly do not have a drive to pursue their future, diluting themeselves far into their present. Segregation of followings was one of few ways to seek motivation beyond the fact."`
			label end
			choice
```

				`	"It seems like you do not like the paths they've chosen."`
			`	With its physique raising above you, the Quarg now stands up, darting its gaze off to the side. The verve it exhibited earlier is now long gone. "It is not in our grasp to persuade the Gegno, as once we did attempt to. They would ensure conflict with us should we interfere, and we have no desire to witness an uprisal of such a sort once more." It pauses, then gestures you to stand up as well. "Similarly, the Gegno do not wish to interact with audacious outsiders. Unfortunately, only a brief history of them we can tell. We cannot advocate pleasantries between you and them, though you are welcome at this ring should you find yourself here again."`
			`	The Quarg motions a bow to you, and the first one from earlier escorts you out of the room. It seems associating with these so-called "Gegno" is very unlikely for you, and you will have to deal with their hostility for the foreseeable future."`
				decline
	on decline

> 		log `Learned from the Quarg that the Gegno are not interested in interaction with outsider species.`

Reviewer 2: Perhaps also add some faction log entries for the Gegno (perhaps as a whole, or both factions separately, or both factions and a more general entry) including some of what the Quarg tells us?

Author: This is done so on offer of the mission.

mission "First Contact: Gegno"
	landing
	source "Giaru Gegno"
	to offer
		has "First Contact: Quarg: offered"
	on offer
		log "Factions" "Gegno" `The Gegno are a humanoid species to the galactic east of the core, split amongst themselves into branches. One branch is the Gegno Vi, who are spoken of to be strong and forceful brutes. The other is the Gegno Scin, mentioned to be progressers of research and technology. Apart from them is a complacent civilian population.`
		conversation
			"After jumping through several systems of hostile alien ships, a Quarg ring world is a sight of relief to you. From the looks of it, this is a fully complete ringworld, a true sight to behold. Trillions of Quarg probably inhabit this ring, but what's more interesting to your eye is that some of the aliens also are present. They are also humanoid, and the most common figures you come across are either slightly above or below an average human's height. Their skin is a smooth, light gray, and adorned with different sorts of leather that remind you of medieval-style human clothing."

> 			`	The Quarg here move in a more eerily-quiet manner than what you're used to, almost dismissive to the other species walking about, who are equally as dismissive of you. The unusual atmosphere around you is finally broken when a few Quarg start to advance towards you, moving much quicker than the rest. After a moment of unidentifiable discussion and gestures between themselves, one of them motions you to follow them.`

Author: Not sure how I feel about this change, but I'll give it some thought.

				goto next
			label sacrifice
			`	"The Gegno believe it to be necessary to sacrifice whatever must be, in order to imagine a future for themselves. Thus, they have separated into two different branches. One brandishes themselves as the honorable Vi - mightiest of warriors, toughest of kin. They are brutes that see proof of strength being the one and only variable in determining their path. That of the Scin, the more technologically able, believe that only through progression of sciences and discovery, will the Gegno prosper."`

			label next
			choice
				`"The Vi are right, the strength to survive is more important than anything else." `
					goto vi
				`"The Scin are right, technological advancement opens up so many possibilities."`
					goto scin
				`"Why do they not combine their efforts to better their future together?"`
					goto together
			label vi

> 			`	"While it is characteristic, such a tremendous proposition is rather dull for a growing species. Be that as it may, you humans follow to a much lesser degree than that of the Vi."`

Author: The idea that war is important in persuading the future, except the humans don't glorify it.

			choice
				`	(Land near the ship and investigate.)`
					goto land
				`	(Continue to search the surface.)`
					goto search
			label land
			`	You carefully land your ship nearby. Most of the faint radar signatures from earlier have disappeared out of range. Exiting your ship in an environment suit and your sidearm on hand, you walk towards the alien ship.`
			`	The ship appears a bit larger than you thought - around the size of a Firebird. The dark-brown hull appears to be made from an unknown material, and has several metallic-looking parts both hatched in crevices and protruding outward. Oddly enough, the warship seems perfectly intact, but the way it is angled into the sand suggests that the ship has been there for some time. There doesn't seem to be any activity around it, and you aren't picking up any nearby life signs. You decide to head back to your ship and take another look around the area.`
			label search
			`	As you fly onward, an erratic signature appears on the edge of your radar. A large arch-like structure becomes visible in the distance, but disappears shortly after. You set your ship down close to the signature and exit to investigate.`

> 			`	Much to your surprise, there is a large humanoid figure standing where the signature originated from. You were sure your sensors showed no life signs, but standing in front of you is a large, bulky being cladded in what looks like a form of metallic armor, wielding a spear-like weapon. The figure moves around attentively as if it is intently focused on its surroundings, but appears to take no interest in your presence. Unlike other aliens you have met in the galaxy, this encounter feels very odd in that there is no immediate interaction with you.`

Author: I'm not sure if that's actually possible - story wise, I think, but in order to get to Gegno space you either need a JD (so you've met aliens at that point that interact with you aka shoot you, or Pug to get the JD in FWs), or you have to fly through Remnant space (where you'll meet the Remnant and possibly Korath or other things). I don't mind taking it out though, for full avoidance.

> 			`	Much to your surprise, there is a large humanoid figure standing where the signature originated from. You were sure your sensors showed no life signs, but standing in front of you is a large, bulky being cladded in what looks like a form of metallic armor, wielding a spear-like weapon. The figure moves around attentively as if it is intently focused on its surroundings, but appears to take no interest in your presence. Unlike other aliens you have met in the galaxy, this encounter feels very odd in that there is no immediate interaction with you.`

Reviewer 3: Yeah I think it's fine.

			`	You should probably go pay the ringworld a visit.`
				accept
				

mission "Giaru Gegno: Revisit"
	name `Visit Giaru Gegno`
	description `Pay the Quarg at Giaru Gegno a visit at the request of the Vi Soldier.`
	landing
	source "Dueyu Eitch"
	destination "Giaru Gegno"
	to offer
		has "First Contact: Quarg: offered"
		has "Giaru Gegno: Quarg Contact: offered"
		has "Visit Quarg in Gegno Space: offered"
	on complete
		conversation
			`	You arrive once more at Giaru Gegno, hoping to find another Quarg that speaks your language. This time, however, you notice that a few of the Gegno on this ringworld occasionally give you a glare of interest as they walk by. Only a minute or so goes by of this, at which a pair of Quarg then approach you, motioning you to follow them. Lead back to the familiar room from before, the Quarg you previously spoke with is already calmly sitting at the other end of the table.`
			

> 			`	"You have astonishingly achieved a wonderous feat, brave traverser," the Quarg says, placing its hands together. The statement alludes to the idea that the Quarg are already aware of your encounter on the alien homeworld. In fact, thinking back to the looks of interest you received earlier, it would seem that the word got spread pretty quickly. "Not significantly often it is the mighty Gegno interact with daring outsiders."`

Author: Are there supposed to be two spaces between `quickly. "Not...`?

> 			`	"You have astonishingly achieved a wonderous feat, brave traverser," the Quarg says, placing its hands together. The statement alludes to the idea that the Quarg are already aware of your encounter on the alien homeworld. In fact, thinking back to the looks of interest you received earlier, it would seem that the word got spread pretty quickly. "Not significantly often it is the mighty Gegno interact with daring outsiders."`

Reviewer 3: No, sorry. Let me fix that.

> 			`	"You have astonishingly achieved a wonderous feat, brave traverser," the Quarg says, placing its hands together. The statement alludes to the idea that the Quarg are already aware of your encounter on the alien homeworld. In fact, thinking back to the looks of interest you received earlier, it would seem that the word got spread pretty quickly. "Not significantly often it is the mighty Gegno interact with daring outsiders."`

Author: I got it don't worry.

			label search
			`	As you fly onward, an erratic signature appears on the edge of your radar. A large arch-like structure becomes visible in the distance, but disappears shortly after. You set your ship down close to the signature and exit to investigate.`
			`	Much to your surprise, there is a large humanoid figure standing where the signature originated from. You were sure your sensors showed no life signs, but standing in front of you is a large, bulky being cladded in what looks like a form of metallic armor, wielding a spear-like weapon. The figure moves around attentively as if it is focused on its surroundings, but appears to take no interest in your presence. Unlike other aliens you have met in the galaxy, this encounter feels very odd in that there is no immediate interaction with you.`
			choice
				`	(Approach the alien.)`
				`	(Turn back to my ship.)`

> 			`	Suddenly, the ground starts to vibrate with increasing intensity. You take out your sidearm in a panic, but as you look back up at the alien figure, you notice that they too enter a defensive stance - but not towards you. A large maw bursts open in the sand to your left, and out of it comes a massive creature hundreds of times the size of you, covered in spikes and scales. You barely have time to react as it plummets onto the sand to your right, but you get a glimpse of the alien creature riding the beast as they impale it with their weapon.`

Author: Suddenly start. My intention was to imply, the ground starts vibrating, and the intensity increases over time. Probably didn't word it very well though.

			label next
			choice
				`	"The Vi are right, the strength to survive is more important than anything else." `
					goto vi
				`	"The Scin are right, technological advancement opens up so many possibilities."`
					goto scin
				`	"Why do they not combine their efforts to better their future together?"`
					goto together
			label vi
			`	"While it is characteristic, such a tremendous proposition is rather dull for a marvelously growing species. Be that as it may, you humans follow this harsh doctrine, though to a much lesser degree than that of the Vi."`
				goto "wrap up"
			label scin
			`	The Quarg now shifts facial expressions in such a way to suggest a small hint of disappointment. "The pursuit of technology leaves behind many values important to a glorious species' growth, that of which we have seen plentifully."`
				goto "wrap up"
			label together
			`	"Regrettably, most Gegno do not have a drive to pursue their principal future, diluting themselves far into their stagnant present. Segregation of followings was one of few ways to seek motivation beyond the fact."`
			label "wrap up"
			choice

> 				`	"It seems that you do not like the paths they've chosen."`

Author: Good idea,

			
			label end
			`	The Quarg motions a bow to you, and the first one from earlier escorts you out of the room. It seems associating with these so-called "Gegno" is very unlikely for you, and you will have to deal with their xenophobia for the foreseeable future.`
				decline

				
mission "Gegno Quarg Direction to Ringworld"
	name `Visit Quarg Ringworld`
	landing
	source
		near "Aleit" 1 2
	destination "Giaru Gegno"
	to offer
		has "First Contact: Quarg: offered"
		not "Giaru Gegno: Quarg Contact: offered"
	on offer
		conversation
			`As soon as you land on this Quarg world, a few Quarg swiftly make their way to your ship. One of them stands in front of the rest and beckons you over while the others quietly speak to themselves in a language you've not heard before. The Quarg bows down slightly to speak to you, and judging by the look in its eyes, they seem rather perplexed. "Sdii veebyi keies Gegnion?" it asks, which seems to be the language the others were speaking in. Afterwords, it patiently waits for your response.`
			choice
				`	(Try to speak to the Quarg.)`

> 			`	You introduce yourself to the Quarg and then explain that you don't understand their language. It lifts itself back up slowly, seemingly pondering the situation before turning back to the others. After a few more exchanged words, one of the other Quarg walk off and fetch some sort of device, bringing it back to you. It appears to be some type of holographic map, and the Quarg gestures to a system nearby, which then reveals a hologram of a Quarg Ringworld. "Ehsveet Duebyuarr Giaru, Aechman," it says, and then bows before you.`

Reviewer 3: ```suggestion
			`	You introduce yourself to the Quarg and then explain that you don't understand their language. It lifts itself back up slowly, seemingly pondering the situation before turning back to the others. After a few more exchanged words, one of the other Quarg walk off and fetch some sort of device, bringing it back to you. It appears to be some type of holographic map. The Quarg gestures to a system nearby, which then reveals a hologram of a Quarg Ringworld. "Ehsveet Duebyuarr Giaru, Aechman," it says, and then bows before you.`
```

			`	You should probably go pay the ringworld a visit.`
				accept
				

mission "Giaru Gegno: Revisit"
	name `Visit Giaru Gegno`
	description `Pay the Quarg at Giaru Gegno a visit at the request of the Vi Soldier.`
	landing
	source "Dueyu Eitch"
	destination "Giaru Gegno"
	to offer
		has "First Contact: Quarg: offered"
		has "Giaru Gegno: Quarg Contact: offered"
		has "Visit Quarg in Gegno Space: offered"
	on complete
		conversation

> 			`	You arrive once more at Giaru Gegno, hoping to find another Quarg that speaks your language. This time, however, you notice that a few of the Gegno on this ringworld occasionally give you a glare of interest as they walk by. Only a minute or so goes by of this, at which a pair of Quarg then approach you, motioning you to follow them. Lead back to the familiar room from before, the Quarg you previously spoke with is already calmly sitting at the other end of the table.`

Reviewer 3: ```suggestion
			`	You arrive once more at Giaru Gegno, hoping to find another Quarg that speaks your language. This time, however, you notice that a few of the Gegno on this ringworld occasionally give you a look of interest as they walk by. After about a minute of this a pair of Quarg approach you, motioning you to follow them. Lead back to the familiar room from before, the Quarg you previously spoke with is already calmly sitting at the other end of the table.`
```

> 			`	"You have astonishingly achieved a wonderous feat, brave traverser," the Quarg says, placing its hands together. It seems the Quarg are already aware of your encounter on the alien homeworld. In fact, thinking back to it, the looks of interest you received earlier suggests that the word got spread quickly. "Not significantly often it is the mighty Gegno interact with daring outsiders."`

Reviewer 3: ```suggestion
			`	"You have astonishingly achieved a wonderous feat, brave traverser," the Quarg says, placing its hands together. It seems the Quarg are already aware of your encounter on the alien homeworld. In fact, thinking back to it, the looks of interest you received earlier suggests that the word spread quickly. "Not significantly often it is the mighty Gegno interact with daring outsiders."`
```

> 			`	"You have astonishingly achieved a wonderous feat, brave traverser," the Quarg says, placing its hands together. It seems the Quarg are already aware of your encounter on the alien homeworld. In fact, thinking back to it, the looks of interest you received earlier suggets that the word got spread quickly. "Not significantly often it is the mighty Gegno interact with daring outsiders."`

Reviewer 3: ```suggestion
			`	"You have astonishingly achieved a wonderous feat, brave traverser," the Quarg says, placing its hands together. It seems the Quarg are already aware of your encounter on the alien homeworld. In fact, thinking back to it, the looks of interest you received earlier suggests that the word got spread quickly. "Not significantly often it is the mighty Gegno interact with daring outsiders."`
```

Author: Whoops. Thanks :P

				accept
			
			label watch
			`	Cautiously retreating towards your ship, you keep your eyes and ears peeled for the creature, wondering if and where it might appear again. As if to answer your inquiry, the ground begins to rumble, and off to your left, sand and rubble erupt into the air as the beast twists and turns across the surface. Following a loud bellow, the creature screeches in short bursts, almost like chirps, except much deeper than that of a bird. As some of the dust clears, you see the alien figure breathing intensely as they continue to battle the beast.`
			choice
				`	(Help the alien.)`
					goto help
				`	(Leave.)`
					goto leave
			label leave
			`	With the situation beginning to spin out of your control, you quickly turn around and make for your ship, taking off as soon as you can. You wonder what will become of the alien figure as while they are most certainly a member of the species inhabiting the nearby systems, it doesn't seem like any help will be sent for them anytime soon.`
				depart
			

> 			label "second encounter"

Author: Me and Derpy took a look at it, and there wasn't really any clean way of doing this, so we left it as-is.

			`	The signatures keep randomly disappearing and reappearing elsewhere, and appear to lead to only empty space. After checking a few of the signatures, an erratic signature appears on the edge of your radar, just like during your first visit. Approaching the location, you recognize the beast that is swinging about on the surface, and catch glimpses of the alien from before continuing to fight it.`
			choice
				`	(Assist the alien in defeating the creature.)`
					goto help
	
	on defer
		set "First Contact: Gegno Vi: deferred"

mission "Visit Quarg in Gegno Space"
	landing
	name `Travel to <designated system>`
	description `Travel to the <designated system> as instructed.`
	substitutions
		"<alien>" "alien"
		"<alien>" "Gegno"
			has "Giaru Gegno: Quarg Contact: offered"
		"<Alien>" "Alien"
		"<Alien>" "Gegno"
			has "Giaru Gegno: Quarg Contact: offered"
		"<aliens>" "aliens"
		"<aliens>" "Gegno"

> 			has "Giaru Gegno: Quarg Contact: offered"

Author: Not sure. I worked this out with Derpy so ask him. He might also change this a bit, who knows.

> 			has "Giaru Gegno: Quarg Contact: offered"

Reviewer 4: Well, for starters, if <aliens> was a global thing, it'd mean that none of the other aliens in the galaxy could use this substitution...

On the flip side, having it be <gegno> that alternates between gegno and aliens depending on the conditions... That makes a lot of sense as a global thing.

> 			has "Giaru Gegno: Quarg Contact: offered"

Reviewer 5: I would prefer them to remain mission-specific, since the replacement name is generic and because it's only these first few missions that need them. Only if the replacement anmes were made less generic (e.g. `<gegno: alien>`) and there were more missions that used them then making them global would make sense.

				launch
			
			label "has met quarg"
			`	After fiddling with it, the Vi marks a new location and presents it to you. "Oukeug," they say, and gesture you to depart in a fashion of urgency. You look at the location and notice that it's the Quarg ringworld you've previously visited. They then leave your ship, walk towards the others, and have a brief discussion. A few moments later, another smaller group of Gegno arrive with what looks to be some type of fueling device, and after a minute or so of fidgeting with it, manage to hook it up to your ship. These Gegno look shorter and thinner than the others, and wear more leather-like garments than heavy metal ones - but their demenaor is just as rigid. Once your ship's fuel has been filled, the Gegno Vi that has been guiding you gestures towards the sky. It seems your visit to the mysterious Gegno homeworld will have to be cut short.`
				launch
	
	npc
		government "Gegno Vi (Neutral)"

> 		personality entering staying

Author: You take off during this mission in said system.

		personality entering staying
		system "Heutesl"
		fleet
			names "gegno scin"
			variant
				"Coesite" 3
				"Felsic" 2
				"Feldspar"
				"Halite" 6
	npc
		government "Gegno Vi (Neutral)"
		personality staying
		system "Heutesl"
		fleet "Vi Warfleet"
	npc
		government "Gegno Scin (Neutral)"
		personality staying	
		system "Heutesl"
		fleet "Scin Warfleet"
	npc
		government "Gegno Scin (Neutral)"
		personality surveillance
		system "Heutesl"
		fleet
			names "gegno scin"
			variant
				"Tridymite"

mission "Giaru Gegno: Quarg Contact"
	landing
	source "Giaru Gegno"
	to offer
		has "First Contact: Quarg: offered"
	on offer
		log "Factions" "Gegno" `The Gegno are a humanoid species to the galactic east of the core, split amongst themselves into branches. One branch is the Gegno Vi, who are spoken of to be strong and forceful brutes. The other is the Gegno Scin, mentioned to be progressers of research and technology. Apart from them is a complacent civilian population.`
		log `Learned from the Quarg that the Gegno are not interested in interaction with outsider species.`
		conversation
			branch "has met gegno"
				has "Visit Quarg in Gegno Space: offered"
				

> 			"After jumping through several systems of many alien ships, a Quarg ring world is a sight of relief to you. From the looks of it, this is a fully complete ringworld, a true sight to behold. Trillions of Quarg probably inhabit this ring, but what's more interesting to your eye is that some other aliens are also present. They too are humanoid, and the most common figures you come across are either slightly above or below an average human's height. Their skin is a smooth, light gray, and adorned with different sorts of leather that remind you of medieval human clothing."

Reviewer 4: ```suggestion
			"After jumping through several systems of many alien ships, a Quarg ring world is a surprisingly familiar sight to you. From the looks of it, this is a fully complete ringworld, a true sight to behold. Trillions of Quarg probably inhabit this ring, but what's more interesting to your eye is that some other aliens are also present. They too are humanoid, and the most common figures you come across are either slightly above or below an average human's height. Their skin is a smooth, light gray, and adorned with different sorts of leather that remind you of medieval human clothing."
```

"Comforting" might be a better word than "familiar", but only if the pilot has a positive relationship with the Quarg...

That aside, it isn't necessarily going to be a relieving sight: If the player has some good ramscoops, or is otherwise comfortable with long-range exploration, they might not be feeling particularly stressed at all.

			`	The Quarg raises from its seat, and you act in accordance. "Please do be distinctly cautious in your encounters with the Gegno. Always feel gracious to visit here once more, should you mindfully chose to do so." As you are directed out of the room, you turn back to see the Quarg bowing. You can't help but wonder what was on their mind during the conversation, but it might be for the best you are more careful from now on.`
				decline

# Gegno Superheavy

mission "Stationed Augen"
	invisible
	landing
	npc save
		government "Gegno Vi"
		personality heroic vindictive staying
		system "Kiluit"
		ship "Augen" "Vytii Yrrlausei"
	to fail
		has "event: Vesvii war"

mission "Augen Destruction"
	landing
	invisible
	to offer
		has "Stationed Augen: failed"
		not "event: Vesvii war"
	on offer
		set "dead augen"
		fail

mission "Active Augen"
	invisible
	to offer
		has "event: Vesvii war"
		has "Stationed Augen: failed"
		not "dead augen"
	npc save
		government "Gegno Vi"
		personality heroic vindictive staying
		system "Vesvii"

> 		ship "Augen" "Vytii Yrrlausei"

Author: Yes, it moves to the capital.

			"reputation: Gegno Vi" < -100
			"reputation: Gegno Scin" < -100
	on offer
		fail "Stationed Augen"
		"reputation: Gegno" = -100
		"reputation: Gegno Vi" = -100
		"reputation: Gegno Scin" = -100
		event "gegno: unified defense"

	npc
		government "Gegno"
		personality nemesis heroic vindictive
		system
			government "Gegno"
		fleet
			names "gegno united"
			variant 
				"Gneiss" 3
				"Hornfel" 2
				"Hornfel (Heavy Weapons)" 2
				"Eclogite" 5
				"Feldspar" 3
				"Halite" 18
				"Felsic" 3
				"Coesite" 5
	npc
		government "Gegno"
		personality nemesis heroic vindictive
		system
			government "Gegno"
		fleet
			names "gegno united"
			variant
				"Feldspar" 2
				"Halite" 12
				"Gneiss" 4
				"Felsic" 4
				"Coesite" 4
				"Eclogite" 3
	npc
		government "Gegno"
		personality nemesis heroic vindictive
		system
			government "Gegno"
		fleet
			names "gegno united"
			variant
				"Feldspar" 4
				"Halite" 24
				"Gneiss" 2
				"Hornfel" 3
				"Hornfel (Heavy Weapons)"
				"Eclogite" 3
				"Coesite" 6
	npc
		government "Gegno Vi"
		to spawn
			has "gegno: unified defense"
		to despawn
			has "dead augen"
		personality heroic vindictive staying
		system "Heutesl"

> 		ship "Augen" "Vytii Yrrlausei"

Author: Yes but that mission moves it, and under those conditions, this one despawns. Ask Derpy, he helped me work these ones out.
